athenahealth/httpclient.go

```diff
@@ -132,6 +132,10 @@ func NewHTTPClient(httpClient *http.Client, practiceID, clientID, secret string)
 
 	noplogger := zerolog.Nop()
 
+	if httpClient.Timeout == 0 {
+		httpClient.Timeout = defaultRequestTimeout
```

Comment:
this change has been superseded by https://github.com/eleanorhealth/go-athenahealth/pull/109/commits/961de65c2b6262f79dc829a62ee3d41edf707b74#diff-bc15f4d80970f6dc182e5a8144c27e27d7f6d3facbbb8f3af3909e0ee5f617b6R100
we don't want to set a timeout on the http client because then we won't be able to increase the timeout beyond that. we need to rely fully on context to allow callers to optionally extend the timeout like we do with image uploads.